packages/turbo-telemetry/README.md

```diff
@@ -0,0 +1,14 @@
+# `@turbo/telemetry`
```

Comment:
Can you include some usage instructions?

Reply:
Yes, will do!